packages/core/index.d.ts

```diff
@@ -14,10 +14,10 @@ export const tools: { env: { inject: (filename?: string) => void } };
 export const createAppTester: (
   appRaw: object,
   options?: { customStoreKey?: string }
-) => <T extends any, B extends Bundle>(
-  func: (z: ZObject, bundle: B) => Promise<T>,
+) => <T, B extends Bundle>(
+  func: (z: ZObject, bundle: B) => T | Promise<T>,
   bundle?: Partial<B> // partial so we don't have to make a full bundle in tests
-) => T extends Promise<T> ? T : Promise<T>;
+) => Promise<T>; // appTester always returns a promise
```

Comment:
This is mind-boggling, took me a while to realize it's a function returning a function. The change makes sense, but I wonder why we need that `bundle: B` part? Why not:

```ts
export const createAppTester: (
  appRaw: object,
  options?: { customStoreKey?: string }
) => <T>(
  func: (z: ZObject, bundle: Bundle) => Promise<T>,
  bundle?: Partial<Bundle> // partial so we don't have to make a full bundle in tests
) => Promise<T>;
```

Unless it's possible to pass in a `Bundle` subclass, i.e., a custom bundle?

Reply:
yeah, great question! 

that was tweaked in response to https://github.com/zapier/zapier-platform-cli/issues/339; 

Basically the default `inputData` is `{[x: string]: any}`, since that's the basic version of what it is. If you pass in a more specific `inputData` interface (which is great, because then `bundle.inputData.x` autocompletes in your editor (and warns you of missing properties)), then TS complained that the more specific type didn't line up with the more generic one. 

`extends` fixes that!